crowdsourcing/models.py

```diff
@@ -559,3 +559,36 @@ class TaskComment(models.Model):
     task = models.ForeignKey(Task, related_name='taskcomment_task')
     comment = models.ForeignKey(Comment, related_name='taskcomment_comment')
     deleted = models.BooleanField(default=False)
+
+
+class FinancialAccount(models.Model):
+    owner = models.ForeignKey(UserProfile, related_name='account_owner', null=True)
+    type = models.CharField(max_length=16, default='general')
+    is_active = models.BooleanField(default=True)
+    balance = models.FloatField(default=0)
```

Comment:
Currency value should be DecimalField rather than FloatField with clear distinction for decimal places.